manifests/base/jupyter-datascience-notebook-imagestream.yaml

```
@@ -42,6 +44,7 @@ spec:
         opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"}]'
         opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.17"},{"name":"Kafka-Python","version":"2.0"},{"name":"Matplotlib","version":"3.4"},{"name":"Numpy","version":"1.19"},{"name":"Pandas","version":"1.2"},{"name":"Scikit-learn","version":"0.24"},{"name":"Scipy","version":"1.6"}]'
         openshift.io/imported-from: quay.io/opendatahub/notebooks
+        opendatahub.io/notebook-build-commit: 
```

```suggestion
        opendatahub.io/notebook-build-commit: 4c8f26e
```

Image:  `quay.io/opendatahub/notebooks:jupyter-datascience-ubi8-python-3.8-4c8f26e`